css/styles.css

```diff
@@ -99,8 +101,122 @@ a {
   background-color: var(--clr-secondary);
 }
 
+/* Learn More */
+
+.movie__image-wrapper {
+  margin: 1rem 1rem 0;
+}
+
+.learn__more-title {
+  font-family: var(--ff-sans);
+  min-height: 65px;
+  padding: 0 1.5rem;
+  margin-bottom: 0;
+}
+
+.learn__more-release {
+  padding: 0 1.5rem;
+  font-family: var(--ff-sans);
+}
+
+.learn__more-release > h3 {
+  margin-top: 0;
+}
+
+.learn__more-summary {
+  padding: 0 1.5rem;
+  line-height: 2rem;
+  font-family: var(--ff-sans);
+}
+
+.learn__more-share {
+  padding: 1rem 1.5rem;
+  gap: 1.5rem;
+}
+
+.learn__more-img {
+  max-height: 300px;
+  object-fit: cover;
+}
+
+.movie__ranking-wrapper {
+  display: none;
+}
+
 @media screen and (width >= 768px) {
   .circle {
     margin-top: 1rem;
   }
 }
+
+@media screen and (width >= 1024px) {
+  .main-wrapper {
+    flex-wrap: nowrap;
+    margin: 2rem;
+  }
+
+  .movie-summary {
+    text-align: end;
+  }
```

Comment:
This could also be a utility class.

Reply:
This is in a media query which I only want to apply in desktop mode.